src/server/worldserver/worldserver.conf.dist

```diff
@@ -1321,6 +1322,55 @@ Warden.BanDuration = 86400
 ###################################################################################################
 
 ###################################################################################################
+# ANTICHEAT SETTINGS
+#
+#     Anticheat.Enable
+#        Description: Enables or disables the Anticheat System functionality
+#        Default:     1 - (Enabled)
+#                     0 - (Disabled)
+
+Anticheat.Enable = 1
```

Comment:
Warden is disabled by default so it makes sense to keep this disabled by default too.